scripts/ui_release.sh

```diff
@@ -74,8 +74,8 @@ function clean() {
   done
 }
 
-function release() {
-  version=${1}
+function bumpVersion() {
```

Comment:
Nit: Should we think about renaming this script to just `release.sh` or something now that it's doing more than just UI-related release tasks?

Reply:
actually instead of renaming I splitted in two different scripts. Just to isolate what is for the ui libs itself and for the release in general.
It's here: https://github.com/perses/perses/pull/374